gen3cirrus/aws/services.py

```diff
@@ -0,0 +1,27 @@
+"""
+Amazon service for interacting with APIs
+"""
+
+import backoff
+import boto3
+from botocore.exceptions import ClientError
+from gen3cirrus.config import config
```

Comment:
I'd prefer if we break up the imports by newlines for 3 sections:

- built in libs
- external deps
- internal imports